index.html

```diff
@@ -136,10 +136,15 @@ <h2>Bronze Sponsors</h2>
   
         <ul class="sponsor__list">
           <li>
-            <a href="https://https://9elements.com" target="_blank" class="sponsor__logo">
+            <a href="https://9elements.com" target="_blank" class="sponsor__logo">
               <img src="./assets/sponsors/9elements_logo_black.svg" alt="9elments company logo">
             </a>
           </li>
+          <li>
+            <a href="https://flexvelop.com" target="_blank" class="sponsor__logo">
+              <img src="./assets/sponsors/flexvelop.svg" alt="flexvelop company logo" style="min-height: 250px;">
+            </a>
+          </li>
```

Comment:
can we vertically center the logo? I guess that would look more clean

<img width="777" alt="2023-website 2023-04-13 17-30-31" src="https://user-images.githubusercontent.com/62679/231810035-2900dd69-9537-4da0-a1d7-3c02ca197117.png">

Reply:
I am not sure how since my CSS skills are limited.
I would therefore merge it now and if someone wants to take a go at it feel free.
But since this is something the sponsors paid for I don't want to keep them waiting to much